src/stream/src/executor/hash_join.rs

```diff
@@ -139,8 +142,9 @@ impl<K: HashKey, S: StateStore> std::fmt::Debug for JoinSide<K, S> {
 }
 
 impl<K: HashKey, S: StateStore> JoinSide<K, S> {
+    // WARNING: Please do not call this until we implement it.
     fn is_dirty(&self) -> bool {
-        self.ht.values().any(|state| state.is_dirty())
+        false
```

Comment:
Should panic here?

Reply:
Maybe implement it after #2999. So we can get if mem table is empty